### PR TITLE
Testing Todo list - 1

### PR DESCRIPTION
**Pull request by Shahid-Bagwan**
following changes were made:

- updated the document.

### CODE REVIEW

**Comment by GhausAliKhan** [comment]
Hi there! use semantic tags & use more comments to make the code more readable

File: src/__test__/add_delete_edit_check.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+});
+
+describe('Delete task', () => {
+  test('Should remove added task', () => {
+    const deleteBtn = document.querySelector('.delete');
+    deleteBtn.click();
+    const listElement = document.querySelectorAll('textarea');
+    expect(listElement).toHaveLength(0);
+  });
+});
```
Comment (shahadat3669) [inline src/__test__/add_delete_edit_check.test.js]: - [x] Well! Your function is working well when I have a single to-do and want to delete it. However, when I have multiple todos, visually I can delete any todo item. In your test function, I can only delete the first one. Please add a condition to check if I can delete other list items when I have multiple to-do items.  Keep up the good work and ensure your functions handle different scenarios appropriately! 👍🧪🚀

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @Shahid-Bagwan, @GhausAliKhan , 

Excellent work thus far! 👏 You've made commendable progress, and you're almost ready for the next project ⏭️, but there are still some minor issues 🐛 that require your attention.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No linter Errors
✅ Descriptive PR Title
✅ Descriptive PR Description
✅ The code is top-notch! 💯


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Thank you for adding the MIT license file to your project. However, I recommend removing all placeholders and updating the license file with the appropriate information. The MIT license requires specific details to be filled in, such as the copyright holder's name and the year of copyright. Please make sure to replace all placeholders with the correct information in the license file. By updating the license file with the correct information, you demonstrate proper adherence to licensing requirements and make it easier for others to understand the terms of using your project. Good luck and keep up the good work! 📝📜🚀

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/add_delete_edit_check.test.js (hunk 1 above)
Comment (Shahid-Bagwan) [reply src/__test__/add_delete_edit_check.test.js]: completed!!

**Review by Strangeal — APPROVED** [review]
Hi @Shahid-Bagwan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**